include/internal/refcount.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+#ifndef HEADER_INTERNAL_REFCOUNT_H
+# define HEADER_INTERNAL_REFCOUNT_H
+
+# if __STDC_VERSION__ >= 201112L && !defined(__STDC_NO_ATOMICS__)
+# include <stdatomic.h>
+# define HAVE_C11_ATOMICS
+# endif
+
+# if HAVE_C11_ATOMICS && ATOMIC_INT_LOCK_FREE > 0
```

Comment:
Well, travis is making no progress, so I attempted --strict-warnings myself, and have got
```
refcount.h:17:23: error: invalid token at start of a
      preprocessor expression
# if HAVE_C11_ATOMICS && ATOMIC_INT_LOCK_FREE > 0
                      ^
```
Do iron it out...